sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -255,6 +255,8 @@ public class Configs {
     private static final boolean DEFAULT_PARTITION_LEVEL_CIRCUIT_BREAKER_DEFAULT_CONFIG_OPT_IN = false;
     private static final String PARTITION_LEVEL_CIRCUIT_BREAKER_DEFAULT_CONFIG_OPT_IN = "COSMOS.PARTITION_LEVEL_CIRCUIT_BREAKER_DEFAULT_CONFIG_OPT_IN";
 
+    private static final String LOAD_AZURE_VM_META_DATA = "COSMOS.LOAD_AZURE_VM_META_DATA";
```

Comment:
Can you please rename this env variable to COSMOS_DISABLE_IMDS_ACCESS to make it consistent with .Net ref. https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4568

Reply:
+1 we have precendence of name fo env variable starting with COSMOS_ while system property starts with COSMOS. and i think this should be the case here as well.

Reply:
Makes sense, let me make it consistent.